tests/backends/tests.py

```diff
@@ -682,6 +682,25 @@ def test_cursor_executemany_with_pyformat_iterator(self):
             self.create_squares(args, 'pyformat', multiple=True)
         self.assertEqual(Square.objects.count(), 9)
 
+    def test_cursor_close_failure_does_not_mask_original_exception(self):
```

Comment:
I'm not sure if it's worth porting this test to master, since Python3 will correctly report chained exceptions?

The exception silencing for `cursor.close()` was only to workaround Python 2 only showing the latest exception (from `cursor.close()`) and not the original exception.